jscomp/Makefile

```diff
@@ -540,11 +540,14 @@ SNAPSHOT_DEPS=$(SNAPSHOT_SRCS:.ml=.d)
 ../lib/bspp.ml:./bin/bspack.exe
 	  $< -D BS_MIN_LEX_DEPS=true -U BS_DEBUG -bs-MD -module-alias Config=Config_whole_compiler   -I $(OCAML_SRC_UTILS) -I $(OCAML_SRC_PARSING)?parser  -I ../lib -I common -I ext -I syntax -I depends -I bspp -I core -bs-main Bspp_main -o $@
 
+../lib/bspp.exe:../lib/bspp.mli ../lib/bspp.ml
+	make -C ../lib bspp.exe
+
```

Comment:
is this rule necessary? seems not to me

Reply:
I was getting and error because `../lib/bspp.exe` is now a dependency of `../lib/reactjs_jsx_ppx_v2.ml` and `bin/jsoo_reactjs_jsx_ppx_v2.ml` so I added that rule (like [this one](https://github.com/astrada/bucklescript/blob/3eb911e17c1077e55aabf9c9fbb5339f163c796e/jscomp/Makefile#L469)). But let me double-check later when I get home.

Reply:
OK, I found the issue. `bspp.exe` is not built by `make world`. It's commented out in `/lib/Makefile` (https://github.com/BuckleScript/bucklescript/blob/master/lib/Makefile#L38). Let me know if you prefer to uncomment it there and remove the rule from this Makefile.